glib-macros/src/lib.rs

```diff
@@ -437,6 +437,8 @@ pub fn closure_local(item: TokenStream) -> TokenStream {
 ///     ValWithCustomNameAndNick,
 /// }
 /// ```
+///
+/// [`glib::Value`]: https://gtk-rs.org/gtk-rs-core/stable/latest/docs/glib/value/struct.Value.html
```

Comment:
This is not useful as you'd point to the stable version of the docs. @GuillaumeGomez @bilelmoussaoui how can this be solved better?

Reply:
as glib-macros doesn't depend on glib itself, i don't see how this could be done without hardcoding one or another version. Unless the `--extern-html-root-url` could help somehow here, not sure.

Reply:
Currently these urls are everywhere in the macro docs, I think it could technically be a link to `../glib/value/struct.Value.html` but I seem to remember there was some issue with that.

Reply:
Should normally work I guess, we do that in the book